### ISSUE TITLE
Wazuh API support for NodeJS 10 and above

### ISSUE THREAD

**Issue by gg-chrisd** [wazuh/wazuh#3134]
Currently `yum install wazuh-api` fails if using NodeJS 10 or 11, I assume due to the format of the versioning changing from 8.x to 0.10.x and 0.11.x, putting it below the nodejs >= 4.6 dependency check.

Not sure if the Wazuh API works with NodeJS 10 or 11, but if it does hopefully it's a fairly simple change to the yum dependency logic to get things up and running.

Thanks

**Comment by crd1985** [comment]
Hi @gg-chrisd,

actually we are working to migrate our API to Python. This change will simplify a lot the Wazuh installation process, removing the wazuh-api package and needing only a single wazuh package. Obviously, the external dependency from NodeJS will also dissappear.

The new API will be released in Wazuh 4.0.0, you can follow the progress of the project [here](https://github.com/wazuh/wazuh/projects/29).

However, we will study how easy is to upgrade the NodeJS version and if it is worth considering the short lifetime remaining to current API version.

Thanks a lot for your suggestion.

**Comment by druizz90** [comment]
Hi @gg-chrisd,

I can run Wazuh API (`3.9` version) with `NodeJS 10` or later. I tested it in a clean environment with `CentOS 7` and `NodeJS 10` and `11`. I used this command for installing `NodeJS` (`10` version):

```bash
# curl -sL https://rpm.nodesource.com/setup_10.x | sudo bash -
```

```bash
node --version
v10.15.3
```

For `11` version:

```bash
# curl -sL https://rpm.nodesource.com/setup_11.x | sudo bash -
```

```bash
# node --version
v11.14.0
```

Could you tell me the steps you did to install `NodeJS` and `Wazuh API`?

Best regards,

Demetrio.
